src/lib/transferExpiryMonitor.js

```diff
@@ -65,9 +68,8 @@ class TransferExpiryMonitor {
         log.debug('transfer ' + transfer.id +
           ' will expire in ' + expiry.diff(now, 'milliseconds') + 'ms')
       }
-    } else if (transfer.state === 'executed' ||
-      transfer.state === 'rejected' ||
-      transfer.state === 'failed') {
+    } else if (transfer.state === transferStates.TRANSFER_STATE_EXECUTED ||
+      transfer.state === transferStates.TRANSFER_STATE_REJECTED) {
```

Comment:
`failed` is no longer a valid transfer state, as per the schema: https://github.com/interledger/five-bells-shared/blob/master/schemas/TransferState.json.